tests/integration/components/aggregate-query-maker-test.js

```diff
@@ -19,36 +19,46 @@ test('it renders', function(assert) {
 
   this.render(hbs`{{aggregate-query-maker startDate="2010-06-01" endDate="2016-07-02" submit=(action mockSubmit) reset=(action mockReset)}}`);
 
-  assert.notEqual(this.$().text(), '');
+  assert.notEqual(this.$().text(), '', 'The component rendered correctly.');
 
 });
 
-/*
-
-test('it submits the query params', function(assert) {
-  const universityVillage = {"type":"Feature","properties":{},"geometry":{"type":"Polygon","coordinates":[[[-87.67248630523682,41.86454328565965],[-87.67248630523682,41.872117384500754],[-87.6549768447876,41.872117384500754],[-87.6549768447876,41.86454328565965],[-87.67248630523682,41.86454328565965]]]}};
-  let mockLayer = L.geoJson(universityVillage);
-
-  this.set('mockSubmit', (actual) => {
-    let expected = {
-      geom: JSON.stringify(mockLayer.toGeoJSON()),
-      startDate: '2016-2-10',
-      endDate: '2016-2-10',
-      agg: 'day'
-    };
-    assert.equal(actual.geoJSON, expected.geom, 'Failed to report correct geoJSON');
-    assert.equal(actual.agg, expected.agg, 'Failed to report selected aggregation unit.');
+test('Parameters correctly control the component.', function(assert){
+  this.set('start', '2010-06-10');
+  this.set('end', '2016-07-20');
+  this.set('agg', 'day');
+  this.set('center', 'chicago');
+
+  this.set('mockSubmit', () => {
+    // NoOp
   });
 
   this.set('mockReset', () => {
     // NoOp
   });
 
-  this.set('layer', mockLayer);
+  this.set('cities', {
```

Comment:
I don't like how we're repeating configuration from the app code in the test code here. When you have the whole shebang in the test, it signals that we needed _all_ of this configuration for the test to pass. Is that so?